scalyr_agent/monitor_utils/k8s.py

```diff
@@ -426,42 +406,8 @@ def _add_to_cache( self, obj ):
             finally:
                 self._lock.release()
 
-    def _process_objects( self, k8s, kind, objects ):
```

Comment:
Seems this function wasn't used anywhere.  Ok to delete?

Reply:
Yes.  This is ok to delete.  It's no longer needed now that we are doing individual queries and no batch queries.